Gemfile

```diff
@@ -49,6 +49,10 @@ gem 'sentry-ruby'
 gem 'sentry-rails'
 gem 'sentry-sidekiq'
 
+gem 'faraday'
+gem 'faraday_middleware'
+gem 'virtus'
```

Comment:
@Jullianos @AndriiMysko `virtus` is discontinued. Use `dry-types`, `dry-struct` and `dry-schema`